<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerForm</class>
 <widget class="QWidget" name="ClientManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTableView" name="clienttableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>phone num</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>address</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>223</width>
         <height>359</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="idLabel">
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="text">
               <string>I&amp;D</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="buddy">
               <cstring>idLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="idLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="nameLabel">
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="text">
               <string>&amp;Name</string>
              </property>
              <property name="buddy">
               <cstring>nameLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="nameLineEdit"/>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QLabel" name="PhoneNumberLabel">
              <property name="text">
               <string>&amp;Phone Number</string>
              </property>
              <property name="buddy">
               <cstring>phoneNumberLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="phoneNumberLineEdit"/>
            </item>
            <item row="12" column="0" colspan="2">
             <widget class="QLabel" name="addressLabel">
              <property name="text">
               <string>&amp;Address</string>
              </property>
              <property name="buddy">
               <cstring>addressLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QLineEdit" name="addressLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>88</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="addHorizontalLayout">
            <item>
             <widget class="QPushButton" name="addPushButton">
              <property name="text">
               <string>&amp;Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="modifyPushButton">
              <property name="text">
               <string>&amp;Modify</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>223</width>
         <height>359</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTreeWidget" name="searchTreeWidget">
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Phone Number</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Address</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchPushButton">
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="text">
             <string>S&amp;earch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
